sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/hashicorp.yaml

```diff
@@ -0,0 +1,32 @@
+provider:
+  metadata:
+    name: Hashicorp
+    category: Cloud provider
+    message: Make sure this Hashicorp token gets revoked, changed, and removed from the code.
+
+  rules:
+    - id: hashicorp-batch-token
+      rspecKey: S6773
+      metadata:
+        name: Hashicorp Batch tokens should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              - b.AAAAAQ
+              - hvb.AAAAAQ
+        matching:
+          pattern: "\\b((?:hv)?b\\.AAAAAQ[\\w-]{120,240})\\b"
```

Comment:
Agree, the `\b` at the end is not adequate. In addition, from what I can see, there is no actual maximum. So maybe, it would be better to remove the maximum. This is what I have done in the update regex,